docs/customized-components/Menu.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'rsg-components/Styled'; // eslint-disable-line import/no-unresolved
```

Comment:
why this ` eslint-disable-line` is necessary here?

Reply:
those components are available however, those are not part of package.json - that's why eslint is angry